modules/caddytls/values.go

```diff
@@ -122,6 +122,7 @@ var SupportedProtocols = map[string]uint16{
 // unsupportedProtocols is a map of unsupported protocols.
 // Used for logging only, not enforcement.
 var unsupportedProtocols = map[string]uint16{
+	//nolint:staticcheck
```

Comment:
What's this about?

Reply:
This is from:

```
SA1019: tls.VersionSSL30 is deprecated: SSLv3 is cryptographically broken, and is no longer supported by this package. See golang.org/issue/32716.  (staticcheck)
```

Obviously we _don't_ support ssl3.0, but it's cringing at the mere mention of it 😉 (and fairly IMO)